src/Microsoft.AspNetCore.Rewrite/Internal/ApacheModRewrite/RuleBuilder.cs

```diff
@@ -18,7 +18,16 @@ public class RuleBuilder
         private UrlMatch _match;
         private CookieActionFactory _cookieActionFactory = new CookieActionFactory();
 
-        private readonly TimeSpan RegexTimeout = TimeSpan.FromMilliseconds(1);
+        private const string UseLowerRegexTimeoutsSwitch = "Switch.Microsoft.AspNetCore.Rewrite.UseLowerRegexTimeouts";
```

Comment:
Move this to a central utility class to reduce duplication.

Reply:
Are you referring to just the string?

Reply:
No, all of the compat switch stuff.